shub/deploy.py

```diff
@@ -1,25 +1,26 @@
 from __future__ import absolute_import
-import os
+
 import glob
+import json
+import os
 import shutil
 import tempfile
-import json
-from six.moves.urllib.parse import urljoin
+from typing import AnyStr, Optional, Union
 
 import click
-import toml
 # Not used in code but needed in runtime, don't remove!
 import setuptools
 import setuptools.msvc  # noqa
+import toml
+from six.moves.urllib.parse import urljoin
 
-from shub.config import (list_targets_callback, load_shub_config,
-                         SH_IMAGES_REGISTRY)
+from shub.config import (SH_IMAGES_REGISTRY, list_targets_callback,
+                         load_shub_config)
```

Comment:
Please put this in one line

Reply:
Sorry, it is an addiction that I have to format the code after making any changes. I have already returned it to the previous state.